meta.yaml

```
@@ -16,39 +16,50 @@ requirements:
   host:
     - gensim >=3.4.0
     - matplotlib >=2.2
-    - networkx >=2.2,<2.4
+    - networkx >=2.2
     - numpy >=1.14
     - pandas >=0.24
     - pip
-    - python
+    - python >=3.6
     - scikit-learn >=0.20
     - scipy >=1.1.0
-    - tensorflow 1.14.*
+    - tensorflow >=2.1.0
     - ipython
     - ipykernel
   run:
     - gensim >=3.4.0
     - matplotlib >=2.2
-    - networkx >=2.2,<2.4
+    - networkx >=2.2
     - numpy >=1.14
     - pandas >=0.24
-    - python
+    - python >=3.6
     - scikit-learn >=0.20
     - scipy >=1.1.0
-    - tensorflow 1.14.*
+    - tensorflow >=2.1.0
     - ipython
     - ipykernel
 
 test:
   imports:
     - stellargraph
+    - stellargraph.connector
+    - stellargraph.connector.neo4j
     - stellargraph.core
     - stellargraph.data
+    - stellargraph.datasets
+    - stellargraph.interpretability
+    - stellargraph.interpretability.saliency_maps
     - stellargraph.layer
     - stellargraph.mapper
     - stellargraph.utils
-    - stellargraph.utils.saliency_maps
-
+  requires:
+    - pytest
+    - pytest-benchmark >= 3.1
+  source_files:
+    - tests
+  commands:
+    - pip install rdflib
+    - py.test -ra tests/ -p no:cacheprovider --benchmark-disable --color=yes
```

This build step takes a long time (12 minutes, according to https://buildkite.com/stellar/stellargraph-public/builds/2857#783ced6d-b185-4363-a480-fe567df8a27b) and most of it is running these tests (8 minutes, https://buildkite.com/stellar/stellargraph-public/builds/2857#783ced6d-b185-4363-a480-fe567df8a27b/167-1184). Do you think it's worth running all of these tests here?

One option would be only doing a subset, as a smoke test, like:

```suggestion
    - py.test -ra tests/core tests/layer -p no:cacheprovider --benchmark-disable --color=yes
```

What do you think?

---

good idea (floated in the original description of this PR that a subset may be more appropriate).

---

Sorry about missing the PR description bit!